### PR TITLE
path/to/binary is the required argument

### PR DESCRIPTION
**Pull request by custa**

### CODE REVIEW

**Comment by derekparker** [comment]
Thanks! I see that you've also updated the docs, but those are autogenerated, and the timestamp should be updated as well. Can you please run `go run ./scripts/gen-usage-docs.go` please?

File: Documentation/usage/dlv.md [hunk 1]
```diff
@@ -41,10 +41,9 @@ Pass flags to the program you are debugging using `--`, for example:
 * [dlv core](dlv_core.md)	 - Examine a core dump.
 * [dlv debug](dlv_debug.md)	 - Compile and begin debugging main package in current directory, or the package specified.
 * [dlv exec](dlv_exec.md)	 - Execute a precompiled binary, and begin a debug session.
-* [dlv replay](dlv_replay.md)	 - Replays a rr trace.
```
Comment (derekparker) [inline Documentation/usage/dlv.md]: Are you on master? This command should not be removed.

**Comment by derekparker** [comment]
@custa ping.

File: Documentation/usage/dlv.md (hunk 1 above)
Comment (custa) [inline Documentation/usage/dlv.md]: I am sorry reply so later.
```go run ./scripts/gen-usage-docs.go``` automatically deletes this command, do I need to manually add it back?

**Comment by derekparker** [comment]
Closing via https://github.com/derekparker/delve/commit/a90893f53284c7a699c0a2f20af326664b5bd1b8 -- thanks!

**Pull request closed without merging**